javaslang/src/test/java/javaslang/collection/AbstractMultimapTest.java

```diff
@@ -354,13 +354,13 @@ public void shouldFindValue() {
 
     @Test
     public void shouldRecognizeNotContainedKeyValuePair() {
-        final TreeMap<String, Integer> testee = TreeMap.of(Tuple.of("one", 1));
+        final Multimap<String, Integer> testee = mapOf("one", 1);
```

Comment:
The tests for `contains` did not use a `Multimap` so the bug did not show in the tests.